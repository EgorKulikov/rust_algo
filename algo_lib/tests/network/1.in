4 6
1 2 1
1 3 1
1 4 2
2 3 1
3 4 1
2 4 1
