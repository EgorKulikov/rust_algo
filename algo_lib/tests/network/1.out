1
4
1 2
1 3
2 3
3 4
